x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/config/Job.java

```diff
@@ -77,7 +77,7 @@
     public static final ParseField MODEL_SNAPSHOT_ID = new ParseField("model_snapshot_id");
     public static final ParseField MODEL_SNAPSHOT_MIN_VERSION = new ParseField("model_snapshot_min_version");
     public static final ParseField RESULTS_INDEX_NAME = new ParseField("results_index_name");
-    public static final ParseField DELETED = new ParseField("deleted");
+    public static final ParseField DELETING = new ParseField("deleting");
```

Comment:
Please can you double check that we have lenient parsing of job configs from cluster state in 6.0.

The reason is that a 6.0 node may end up parsing a 6.5 job that is being deleted if a full cluster restart of a mixed version cluster happens for some reason.  Assuming we have lenient parsing for stored job configs in 6.0 this should work as before - the fact that the job was being deleted when the cluster was shut down will be forgotten, but it can be deleted again - same as it always was in 6.0.

Reply:
I checked and there is lenient parsing in `6.0` so we're good on this front.